u-boot/cpu/mips/ar7240/qca_dram.c

```diff
@@ -624,6 +629,11 @@ static inline void qca_dram_set_ddr_cfg2(u32 mem_cas,
 
 	/* Gate open latency = 2 * MEM_CAS */
 	tmp = 2 * mem_cas;
+#if (SOC_TYPE & QCA_AR933X_SOC)
```

Comment:
On more thing, full details of the RAM chip (manufacturing info): -

Zental
A3R12E40CBF-AH   
524ZENA8
5605 494G-F

this is 512Mb (64MB), specification states AH = DDR2-1066(7-7-7)